core/common/src/DisplayStyleSettings.ts

```diff
@@ -78,6 +78,13 @@ export interface OrbitGtBlobProps {
   accountName: string;
 }
 
+/** JSON representation of the attribution properties for a reality model. These are used to create the logo card for the model using [IModelApp.makeLogoCard]($frontend).
+ * @beta
+ */
+export interface RealityModelAttribution {
+  heading: string;

```

Comment:
Please describe how each gets displayed by the logo card, or link to `makeLogoCard`.